test/jdk/java/awt/print/Dialog/DialogOwnerTest.java

```diff
@@ -98,7 +98,8 @@ public static void main(String[] args) {
        "window, or on top of all windows.\n" +
        "For Owned tests the window titled 'Owner Window' should always \n" +
        "stay behind the print dialog.\n" +
-       "For On Top tests all windows should stay behind the owner window.\n" +
+       "For On Top tests the print dialog should stay above \n" +
+       " the window titled 'Owner window'.\n" +
```

Comment:
Hmm. What did I mean here ?
I think it should say
"For On Top tests all windows should stay behind the print dialog".
Where all windows means all windows on the desktop, not just the Java ones.

Reply:
OK..That's what I presumed..Fixed as per your version.